examples/cndp-frr/docker/frr-scripts/startup.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) Red Hat Inc.
+
+sysctl -p /etc/sysctl.d/90-routing-sysctl.conf
```

Comment:
I did not see any sysctl that stopped the forwarding, is that something we need?

Reply:
This script runs inside the container, so when the container is gone it's effects are also gone (it's scope is the lifetime of the container).